RiotSharp.Test/RiotApiTest.cs

```diff
@@ -282,7 +282,7 @@ public void GetLeaguesAsync_BySummoner_Test()
             {
                 var leagues = api.GetLeaguesAsync(RiotApiTestBase.summonersRegion, RiotApiTestBase.summonerIds.FirstOrDefault());
 
-                Assert.IsTrue(leagues.Result.Count > 0);
+                Assert.IsNotNull(leagues);
```

Comment:
I'm pretty unsure of this assert, that's just a test i think i will remove it from the commit.